### PR TITLE
Docs: Rename Libraries to SDKs

### PR DESCRIPTION
**Pull request by Ekwuno**
This PR updates the current language support to be SDKs since we essentially have the same methods for implementation. Also ensures that  when we have framework integrations we aren't mixing them up.

### CODE REVIEW

File: app/components/section/features/index.hbs [hunk 1]
```diff
@@ -1191,38 +1191,38 @@
 						Javascript
 						<l green>Available</l>
 					</h4>
-					<p>A real-time, fast SDK library for Javascript with bi-directional communication over WebSockets.</p>
+					<p>A real-time, fast SDK SDK for Javascript with bi-directional communication over WebSockets.</p>
```
Comment (kearfy) [inline app/components/section/features/index.hbs:1194]:
```suggestion
					<p>A real-time, fast SDK for Javascript with bi-directional communication over WebSockets.</p>
```

@Ekwuno `SDK` is duplicate in a lot of places

**Review by kearfy — APPROVED** [review]
LGTM!

**Comment by kearfy** [comment]
Actually before you merge, can we add redirects for the renamed routes?

**Comment by Ekwuno** [comment]
Do we have redirects in the docs? @kearfy

**Comment by kearfy** [comment]
Have a look at this file: https://github.com/surrealdb/www.surrealdb.com/blob/main/app/routes/discord.js

**Pull request merged**